jsf/subsystem/src/test/java/org/jboss/as/jsf/deployment/JSFModuleIdFactoryTestCase.java

```diff
@@ -32,15 +32,15 @@
 public class JSFModuleIdFactoryTestCase {
 
     private static final String API_MODULE = "jakarta.faces.api";
-    private static final String IMPL_MODULE = "com.sun.jsf-impl";
+    private static final String IMPL_MODULE = "jakarta.faces.impl";
     private static final String INJECTION_MODULE = "org.jboss.as.jsf-injection";
 
     private static final JSFModuleIdFactory factory = JSFModuleIdFactory.getInstance();
 
     @Test
     public void getActiveJSFVersionsTest() {
         List<String> versions = factory.getActiveJSFVersions();
-        Assert.assertEquals(2, versions.size());
+        Assert.assertEquals(3, versions.size());
```

Comment:
I'm not going to hold up merging because of this, but it would be good if some of the checks removed in WFLY-18055 were restored:

1) Assert.assertTrue(versions.contains("myfaces2")); // Explains why the expected count is 3, not two
2) Assert.assertFalse(versions.contains("bogus")); // Confirms that only slots with an api, impl and injection module get counted

Before WFLY-18055, the assertFalse was for "1.2" but that's gone. But there's still a "bogus" that can be used.